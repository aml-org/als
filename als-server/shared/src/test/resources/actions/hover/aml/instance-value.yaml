#%MetaHover 1.0
rootName: name
version: vers[*]ion
son:
  path1:
    kind: 1
    description: one
  path2:
    kind: 2
    description: two
