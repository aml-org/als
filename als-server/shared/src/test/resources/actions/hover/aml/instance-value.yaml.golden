hovers:
  (3,13):
    range: Range(common.Position[L:2,C:0],common.Position[L:3,C:0])
    contents:
      - The version of the root element and the hole document
